server/common/model/src/test/java/com/alipay/sofa/registry/common/model/metaserver/NodeServerOperateInfoTest.java

```diff
@@ -75,4 +80,24 @@ public void test() {
+    int times = 100;
+    long start = System.currentTimeMillis();
+    for (int i = 0; i < times; i++) {
```

Comment:
Use @BenchmarkMode to create a benchmark test will be better than a unit test,  the benchmark test will output a benchmark result with cpu, memory, cost time in a report.